Tests/Message/Handler/WarmupCacheHandlerTest.php

```diff
@@ -36,13 +33,6 @@ protected function setUp(): void
         }
     }
 
-    public function testShouldImplementMessageHandlerInterface(): void
-    {
-        $rc = new \ReflectionClass(WarmupCacheHandler::class);
-
-        $this->assertTrue($rc->implementsInterface(MessageHandlerInterface::class));
-    }
-
```

Comment:
Since the interface is removed from the framework, and the class no longer implements it, the test is unnecessary.